practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/service/EntityService.java

```diff
@@ -20,5 +21,14 @@
 
     String saveRoleAssignment(RoleAssignment roleAssignment);
 
```

Comment:
Can this interface be removed? There is only one service in this module, the practitioner service. This interface isn't general enough to be useful (even if there was other code getting reference to the practitioner service, which there shouldn't be apart from tests).

Reply:
I prefer not to change the module too much at this stage. Lets discuss offline.